docs/pages/setup/reference/networking.mdx

```diff
@@ -69,6 +72,14 @@ Environment="NO_PROXY=localhost,127.0.0.1,192.168.0.0/16,172.16.0.0/12,10.0.0.0/
 ```
 
 When Teleport builds and establishes the reverse tunnel to the main cluster, it will funnel all traffic through the proxy. Specifically, if using the default configuration, Teleport will tunnel ports `3024` (SSH, reverse tunnel) and `3080` (HTTPS, establishing trust) through the proxy.
+If you need to not proxy some of this traffic (for example, proxying HTTPS but not SSH), add the corresponding `host:port` to `NO_PROXY`.
+
+```
+[Service]
+Environment="HTTP_PROXY=http://proxy.example.com:8080/"
+Environment="HTTPS_PROXY=http://proxy.example.com:8080/"
+Environment="NO_PROXY=example.com:3024"
```

Comment:
Couple of questions here:

- Maybe say `teleportproxy.example.com` to indicate that this is address of the Teleport Proxy Service?
- Is it enough to just include proxy's web endpoint here? Have we tested this? I seem to remember a customer who encountered this before we rolled back the change in v9 had to add `kube.teleport.cluster.local` to NO_PROXY. Is it not needed?

Reply:
`kube.teleport.cluster.local` is needed; I added the docs for it in #13048.